splitio/sync/split.py

```diff
@@ -19,7 +23,7 @@
 
 
 _ON_DEMAND_FETCH_BACKOFF_BASE = 10  # backoff base starting at 10 seconds
-_ON_DEMAND_FETCH_BACKOFF_MAX_WAIT = 60  # don't sleep for more than 1 minute
+_ON_DEMAND_FETCH_BACKOFF_MAX_WAIT = 30  # don't sleep for more than 1 minute
```

Comment:
why we change this? comment should be 30 secs instead of 1 minute

Reply:
I am using the same constant for both sync.split.py and sync.segment.py, I think it can be the same amount for both.

Reply:
will fix the comment.